### ISSUE TITLE
Register `perfit.fedimint.org` and point at our instance

### ISSUE THREAD

**Issue by dpc** [fedimint/fedimint-infra#7]
```
> ping perfit-fedimint.dpc.pw
PING perfit-fedimint.dpc.pw (65.109.78.39) 56(84) bytes of data.
^C64 bytes from 65.109.78.39: icmp_seq=1 ttl=47 time=177 ms
```

I used own domain just to get going.

@elsirion

**Comment by elsirion** [comment]
I made it `perfit.dev.fedimint.org` so we can migrate the entire `dev.fedimint.org` namespace to DNS servers managed in this repo.

**Comment by elsirion** [comment]
Closed by #8
